VogtPayroll4/Employee.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace VogtPayroll4
+{
+    class Employee
+    {
+        public string empName;
+        public string empID;
```

Comment:
only the employee name should be a string.

I'd make the rest of those ints to start.